resources/views/components/footer.blade.php

```diff
@@ -35,13 +35,13 @@
 <div {{ $attributes->class('flex w-full flex-col gap-2') }}>
     <div class="flex w-full justify-center">
         <img
-            class="hidden h-5 dark:block"
-            src="{{ Vite::asset('resources/images/canyon-logo-dark.png') }}"
+            class="hidden h-7 dark:block"
+            src="{{ Vite::asset('resources/images/canyon-logo-dark.svg') }}"
             alt="{{ config('app.name') }}"
         />
         <img
-            class="block h-5 dark:hidden"
-            src="{{ Vite::asset('resources/images/canyon-logo-light.png') }}"
```

Comment:
Are these old PNGs used or referenced anywhere else in the app? If not, let's please delete them.